gen/tests/test_adminrouter_tls_conf.py

```diff
@@ -12,15 +12,18 @@ class TestAdminRouterTLSConfig:
     Tests for the Admin Router TLS Config creation.
     """
 
-    def test_default(self):
+    @pytest.mark.parametrize(
```

Comment:
These tests do not cover what happens when changing `adminrouter_tls_1_1_enabled` or `adminrouter_tls_1_2_enabled`.

Reply:
Or maybe this is covered by `test_enable_custom_tls_versions`, in which case, either add a comment or remove the parametrisation.